hacking/ticket_stubs/bug_internal_api.md

```diff
@@ -1,16 +1,23 @@
-Internal Ansible API
-====================
-
 Hi!
 
 Thanks very much for your submission to Ansible.  It sincerely means a lot to us
```

Comment:
When you mention sincerity or honesty, you often suggest its opposite in the mind of the reader. This is a change I would suggest throughout. 
```suggestion
Thanks very much for your submission to Ansible.  It means a lot to us
```

Reply:
Yup, I was wondering about that as well, thanks Alicia